src/recipes/modalDescriptionClass.tsx

```diff
@@ -0,0 +1,9 @@
+import { css } from '../../styled-system/css'
+
+// Overrides the default paragraph margin in the modal recipe.
+const modalDescriptionClass = css({
+  fontSize: '18px',
+  marginBottom: '30px',
+})
+
+export default modalDescriptionClass
```

Comment:
Can this be renamed to `modalDescriptionClass.ts`? The `.tsx` extension should only be required for files that use the JSX syntax.